docs/data/sources/segment.md

```diff
@@ -5,24 +5,32 @@ description: By tracking events and users via Segment's API and libraries, you c
 
 With Segment, you can manage data and integrations with services across your Growth, Product, and Marketing stack. By tracking events and users via Segment's API and libraries, you can send your product's data to all your analytics and marketing platforms, with minimal instrumentation. They offer support for most platforms, including iOS, Android, JavaScript, Node.js, PHP, and more.
 
-This guide complements [Segment's integration documentation](https://segment.com/docs/integrations/amplitude/). It provides more details on the different types of Segment integrations you can use, how they affect your data in Amplitude, and instructions for setting up the integrations.
+This guide complements [Segment's Amplitude (Actions)](https://segment.com/docs/connections/destinations/catalog/actions-amplitude/) documentation. It is most up-to-update with Segment, provides more details on how they affect your data in Amplitude, and instructions for setting up the integrations.
 
-!!!note "Other Amplitude + Segment integrations"
-
-    This integration imports Segment data into Amplitude. Amplitude offers other integrations with Segment: 
+!!!warning "Segment's Amplitude (Classic)"
```

Comment:
Can we have a link for `Segment's Amplitude (Actions)`. And we can say -
This document is mainly focused on the new destination, "Segment's Amplitude (Actions)" instead of "Segment's Amplitude (Classic)"